common/components/enums/Section.js

```diff
@@ -31,6 +31,7 @@ const Section = {
   VideoOverview: "VideoOverview",
   AboutEventProject: "AboutEventProject",
   CreateEventProject: "CreateEventProject",
+  Privacy: "Privacy",
```

Comment:
Note that enums/Section.js needs to be kept in sync with the helpers/constants.py file, so match the entries in both places.